metadata/src/main/java/edu/unc/lib/dl/util/DepositStatusFactory.java

```diff
@@ -4,9 +4,12 @@
 import static edu.unc.lib.dl.util.RedisWorkerConstants.DEPOSIT_STATUS_PREFIX;
 import static edu.unc.lib.dl.util.RedisWorkerConstants.INGESTS_CONFIRMED_PREFIX;
 import static edu.unc.lib.dl.util.RedisWorkerConstants.INGESTS_UPLOADED_PREFIX;
+import static edu.unc.lib.dl.util.RedisWorkerConstants.DEPOSIT_MANIFEST_PREFIX;
```

Comment:
Good catch. Not sure why git didn't pick up that file, but I re-committed and push out the change,